src/coins.h

```diff
@@ -31,14 +31,20 @@ class CBlockUndo;
  * - unspentness bitvector, for vout[2] and further; least significant byte first
```

Comment:
formatting of comments: '-' is used as paragraph marker, remove it if the phrase contonues

Reply:
Done